install_prerequisites/build

```diff
@@ -72,7 +72,7 @@ set_default_version()
   # See http://stackoverflow.com/questions/1494178/how-to-define-hash-tables-in-bash
   package_version=(
     "cmake:3.4.0"
-    "gcc:trunk"
+    "gcc:gcc-5-branch"
     "mpich:3.1.4"
```

Comment:
@rouson So this will always download the latest gcc-5.x stable release? Or does it download the latest 5.x release candidate/bleeding edge?

Are any other download dependencies using the `svn` fetch method? It might be worth considering other fetch methods for GCC just to remove a dependency (or pseudo/meta dependency) on subversion. Thoughts are:

 1. Download a release tarball with `wget` or `curl` from one of the [GCC mirrors](https://gcc.gnu.org/mirrors.html)
     - `http://ftpmirror.gnu.org/gcc/` will resolve to the geographically closest mirror for downloading
 2. The GCC subversion repository is mirrored via `git-svn` at https://github.com/gcc-mirror/gcc/branches; so if a user doesn't have

This is just food for thought, and this seemed like a decent place to ask you about it.

I'd be happy to add improving some logic surrounding the `fetch` method to my "someday" todo list.

 - Could query the system when first running the script to see if wget, curl or ftp is present, and then where appropriate use one of these download methods.
 - I think subversion repos can sometimes be fetched with these tools as well (if they are webdav based) but I don't remember for sure
 - Could check in the beginning for `svn` and `git`

Any way none of this is urgent at all. If this is of any interest to you we can open up an issue to document and discuss what we would like to see happen.